components/navbar/sample.html

```diff
@@ -0,0 +1,97 @@
+    Donec venenatis mattis erat ut semper. Aenean id nisl accumsan, tempus enim quis, accumsan lectus. Vivamus rutrum egestas quam, ut lobortis nibh tristique id. Etiam gravida at metus quis egestas. Curabitur ultricies a ipsum ut lacinia. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Phasellus suscipit metus urna, ut auctor risus imperdiet vitae. Etiam in mollis est. Nulla ultricies lectus nisl, vitae fringilla nisl venenatis non. Vivamus tincidunt cursus purus et facilisis.
+  </p><p style="margin-left: 25px; margin-right: 25px;">
+    Donec ac dui ex. Nulla est leo, vehicula vel hendrerit vel, ornare id erat. Mauris eget dignissim erat, sit amet pretium enim. Donec tincidunt dolor vitae libero ullamcorper molestie. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec lacinia hendrerit dapibus. Maecenas mollis lorem sit amet facilisis feugiat. Sed faucibus lectus et blandit dictum. Mauris augue purus, placerat quis fermentum et, ullamcorper sit amet felis. Vivamus at ultricies odio, ac condimentum diam. Praesent faucibus quis tellus et maximus.
+  </p><p style="margin-left: 25px; margin-right: 25px;">
+    Donec accumsan tincidunt neque, non interdum eros venenatis eget. Phasellus condimentum convallis purus, a gravida diam dignissim nec. Nulla convallis magna ut ligula luctus, at ultricies nisi lacinia. Nullam ornare sodales lobortis. Suspendisse mattis commodo felis, id tempus risus dapibus ac. Phasellus non hendrerit dolor. Integer ac arcu at elit tempus condimentum non sed metus. Nunc egestas eros non imperdiet accumsan. Sed molestie tortor in imperdiet ornare. Vivamus malesuada blandit augue, ut euismod nisi mollis molestie. Phasellus porttitor ex non nulla commodo, sit amet efficitur nisi lacinia.
+  </p><p style="margin-left: 25px; margin-right: 25px;">
+    Sed consectetur egestas lectus sed hendrerit. Donec molestie orci quis erat scelerisque aliquet. Sed auctor maximus feugiat. Proin mollis, risus at dapibus dapibus, dolor metus volutpat sapien, sed condimentum magna turpis non purus. Pellentesque auctor felis eget rhoncus pellentesque. Sed neque enim, imperdiet vitae egestas placerat, commodo sed mauris. Etiam egestas lacus ac ex commodo laoreet. Suspendisse sed augue vitae ex elementum hendrerit. Pellentesque ullamcorper facilisis augue, ut lacinia ipsum consectetur sit amet. Aenean eget diam ac neque scelerisque placerat.
+  </p>
+
+  <script>
+    const useCapture = false;
+    const show = elem => elem.style.display = 'block';
+    const hide = elem => elem.style.display = 'none';
+    const toggle = (elem) => {
+      if (window.getComputedStyle(elem).display === 'block') {
+        hide(elem);
+        return;
+      }
+
+      show(elem);
+    };
+
+    // Listen for click events
+    document.addEventListener('click', (event) => {
```

Comment:
was there a reason to bind this globally?
i'd change this to specifically bind to an element instead of the entire document.
```
const menuButton = document.getElementById('menu');
menuButton.addEventListener('click', ... 
```

Reply:
Fixed.